### PR TITLE
Add info on WebSocket

### PR DESCRIPTION
**Pull request by Maria-Nosyk**

### CODE REVIEW

File: src/advanced/node-management.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 <!-- cspell:ignore nanos -->
 
-Exonum nodes can be controlled using RPC implemented via REST API. Managing
-endpoints are handled by Exonum core and are mainly purposed to receive
+Exonum nodes can be controlled using RPC implemented via REST API and WebSocket.
```
Comment (aleksuss) [inline src/advanced/node-management.md]: I don't think that we can control exonum node via WebSocket. We can get some piece of information via WebSocket. And that all.

File: src/advanced/network.md [hunk 2]
```diff
@@ -32,12 +32,18 @@ Full nodes are further subdivided into 2 categories:
...
-to full nodes to retrieve information from the blockchain they are
-interested in, and to send transactions. Exonum provides a
-[“proofs mechanism”](../glossary.md#merkle-proof),
+to full nodes for several purposes:
+
+- to [retrieve](node-management.md#explorer-api-endpoints) information they are
+  interested in from the blockchain
+- to [subscribe](node-management.md#explorer-api-sockets) to block commit events
+  and be aware of every new accepted block
+- to send transactions.
```
Comment (aleksuss) [inline src/advanced/network.md]: If we get an information about new block committing we can not only send transactions. I'd re-write it: 
`to subscribe to events and be aware of every new block accepted.`

File: src/advanced/network.md [hunk 3]
```diff
@@ -32,12 +32,18 @@ Full nodes are further subdivided into 2 categories:
     _light clients_.
 
 **Light clients** represent clients in the client-server paradigm; they connect
-to full nodes to retrieve information from the blockchain they are
-interested in, and to send transactions. Exonum provides a
-[“proofs mechanism”](../glossary.md#merkle-proof),
+to full nodes for several purposes:
+
+- to [retrieve](node-management.md#explorer-api-endpoints) information they are
+  interested in from the blockchain
```
Comment (aleksuss) [inline src/advanced/network.md]: Nit: dot missing ?

File: src/advanced/node-management.md [hunk 4]
```diff
@@ -755,6 +755,64 @@ equal to `start` and is less than `end`.
...
     range from `288` to `6` has been traversed.
 
+## Explorer API Sockets
+
+Since Exonum 0.10 version, it is possible to connect to nodes via WebSocket.
+
+Explorer API sockets have the same base path as endpoints, denoted
+**{explorer_base_path}** and equal to `/api/explorer/v1`.
+
+Currently only one socket is implemented - it shares information on block
```
Comment (aleksuss) [inline src/advanced/node-management.md]: `Currently only one socket is implemented` -> `Currently only one type of events is provided`

File: src/advanced/node-management.md [hunk 5]
```diff
@@ -755,6 +755,64 @@ equal to `start` and is less than `end`.
...
+
+### Subscribe to Block Commit
+
+```none
+ws://${URL}{explorer_base_path}/blocks/subscribe
+```
+
+Connects to a socket and receives notices on each new committed block starting
+from the moment of connection. The notices are displayed in the blockchain
+explorer.
```
Comment (aleksuss) [inline src/advanced/node-management.md]: > The notices are displayed in the blockchain explorer.
Is it really true ?

File: src/advanced/node-management.md [hunk 6]
```diff
@@ -755,6 +755,64 @@ equal to `start` and is less than `end`.
...
+#### Parameters
+
+None.
+
+#### Response
+
+Returns notifications that a new block has been committed to the blockchain
+starting from the height when the client connected to the socket.
+
+Each notification is a JSON object with the following fields:
```
Comment (aleksuss) [inline src/advanced/node-management.md]: `Each notification is a string witch could be deserialized into JSON object which contains the following fields:`

File: src/advanced/node-management.md (hunk 1 above)
Comment (Maria-Nosyk) [reply src/advanced/node-management.md]: updated

File: src/advanced/network.md (hunk 2 above)
Comment (Maria-Nosyk) [reply src/advanced/network.md]: I do not get your point. Light clients have 3 different purposes. Subscription to block commit and sending transactions are different purposes. I mention them separately. Besides, note that "send transactions" was mentioned even before I added subscription to events.

File: src/advanced/network.md (hunk 3 above)
Comment (Maria-Nosyk) [reply src/advanced/network.md]: each point here is a continuation of the sentence "**Light clients** represent clients in the client-server paradigm; they connect to full nodes for several purposes:"
The dot should be put only in the end of the sentence, i.e. after the last point only.

File: src/advanced/node-management.md (hunk 4 above)
Comment (Maria-Nosyk) [reply src/advanced/node-management.md]: done

File: src/advanced/node-management.md (hunk 5 above)
Comment (Maria-Nosyk) [reply src/advanced/node-management.md]: done

File: src/advanced/node-management.md (hunk 6 above)
Comment (Maria-Nosyk) [reply src/advanced/node-management.md]: done

File: src/advanced/network.md (hunk 2 above)
Comment (aleksuss) [reply src/advanced/network.md]: I got it. It's ok. 👍🏻 I read it as one sentence because a dot is absent after `... accepted block`.

**Pull request merged**